### DISCUSSION TITLE
Difference in bed tramming between probe and manual metering

### DISCUSSION THREAD

**Discussion by grrmisfit** [mriscoc/Ender3V2S1#53] [General]
the back left corner ( spot where bed wires come in ) is always off. rest of the bed can be almost level with minor variations but that back left is always off.  Probe tramming also gives considerable worse measurements vs manual. After doing a manual tramming most mesh points are .02 to .06 with back left being .24. This is also being reported on facebook S1 owners group https://www.facebook.com/groups/602529180853579/permalink/626391038467393/

**Comment by mriscoc** [comment]
That could be an indicative of cross talk (electrical noise) due to the heating bed, I suggest you increase the probe margin to move away the probing points from that zone.

**Reply by georgesir** [reply]
Hi Mriscoc, 

I have the same problem with a variance of 0.2 to 0.3 on the right side of my bed, where the printing will occur in the air. Any idea how to fix it? Or if possible can you add a manual adjustment for the mesh level prob values? 

Thank you for all your hard work and would love to contribute, where can i?

**Reply by mriscoc** [reply]
> Hi Mriscoc,
> 
> I have the same problem with a variance of 0.2 to 0.3 on the right side of my bed, where the printing will occur in the air. Any idea how to fix it? Or if possible can you add a manual adjustment for the mesh level prob values?
> 
> Thank you for all your hard work and would love to contribute, where can i?

Hi, for correct a point of the mesh you can use the g-code command M421: https://marlinfw.org/docs/gcode/M421.html
At the top of the page there is a `Sponsor` link, thank you.

**Reply by grrmisfit** [reply]
> That could be an indicative of cross talk (electrical noise) due to the heating bed, I suggest you increase the probe margin to move away the probing points from that zone.

according to reading the cr-touch uses an optical sensor which doesnt suffer from interference like the Bl-touch does. ive tested this on 2 machines (S1) and both have same issue. To further make sure it wasnt intereference incase the article was wrong, i covered the spots and cable with foil and get same results

**Reply by mriscoc** [reply]
Yes, the tip sensor of the CRTouch is optical while the 3D/BLTouch uses Hall effect, but the crosstalk is not about the sensor itself, it is about the wires, that is the reason because sometimes twisted the wires or put a ferrite ring helps. In any case the problem could be in a corner but not along the left side as I have seen in some pictures. I recently asked some users that have the S1 and not all have the same failure with CRTouch despite using the same version of the firmware, then the fault is something yet to be discovered.

**Comment by mriscoc** [comment]
Please can you use the G30 command to check the problematic mesh points? I don't have the printer so I need your feedback to try to understand what is failing. Is the stock firmware probing at the same places?

**Comment by severandynasty** [comment]
I'll also tag on there that I'm seeing an oddity in my mesh.  I'm running a 5x5 and the right most column (front to back, right edge of bed) is showing a serious discrepancy.  I have manually trammed the bed with a .10 feeler gauge and am confident it is in a good place but my mesh all of a sudden shows serious dips on the right side and my first layer is too far away on that side but is fine in the left and center section.

Here is the current mesh after a manual tram:

![PXL_20220225_212142504.MP.jpg](https://user-images.githubusercontent.com/57111669/155804691-f7838217-dcb2-46dc-a99b-592012964d96.jpg)
